cmd/apprepository-controller/controller_test.go

```diff
@@ -13,20 +13,44 @@ import (
 )
 
 func Test_newCronJob(t *testing.T) {
-	dbURL = "postgresql.kubeapps"
-	dbName = "assets"
-	dbUser = "admin"
-	dbSecretName = "postgresql"
-	const kubeappsNamespace = "kubeapps"
+	var kubeconfig = ""
```

Comment:
just for styling, it's more common to use `kubeconfig := ""` rather than `var kubeconfig = ""`. The `var` notation is typically only used when you don't know the initial value (or if you don't want to set it) e.g. `var kubeConfig *kube.Config`